### DISCUSSION TITLE
Unable to Login

### DISCUSSION THREAD

**Discussion by Vivekagent47** [vercel/community#2056] [Help] [answered]
Whenever I am trying to login in Vercel with my GitHub account, it asks me for my phone number but when I enter it, it shows me an error that the phone number is already associated with Vercel account, and I am unable to login.

<img width="801" alt="Screenshot 2023-04-13 at 11 42 57" src="https://user-images.githubusercontent.com/48582432/231670078-df64c342-0981-4878-8976-7cadaebf18f3.png">

**Comment by swarnava** [comment, marked as answer]
Please contact us in registration@vercel.com so we can take a look.
